extensions/cornerstone/src/components/StudySummaryMenu.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { StudySummaryFromMetadata } from './StudySummaryFromMetadata';
+import StudyMeasurementMenu from './StudyMeasurementMenu';
+
+export default function StudySummaryMenu(props) {
```

Comment:
What does "Menu" mean in this `StudySummaryMenu`? Shouldn't it be called StudySummaryWithExports or something like that?